test/log_subscriber_test.rb

```diff
@@ -10,7 +10,7 @@ class LogSubscribersController < ActionController::Base
 
     def with_page_cache
       cache_page('Super soaker', '/index.html')
-      render nothing: true
+      render body: nil
```

Comment:
@eileencodes Sure thing! The deprecation message mentioned either `body: nil` or `head :ok` would work in lieu of `nothing: true` - but I will opt for the latter. Either one broke Rails 4.0 anyways.